Backpack/BarChart/README.md

```diff
@@ -1,4 +1,151 @@
 # Backpack/BarChart
 
-## TODO
+## Installation
+
+In `Podfile` add
+
+```
+pod 'Backpack/BarChart'
+```
+
+and then run `pod install`.
+
+## Usage
+
+### Bar chart
+
+`Backpack/BarChart` is a component for presenting data to travellers.
```

Comment:
And the prize for 'most generic component description' goes to... 😂 

It doesn't matter because they'll be able to tell from the name, but still, this could describe literally every component 😝